### PR TITLE
Redirect intro.html to index.html

### PR DESCRIPTION
**Pull request by richvdh**
Fixes https://github.com/matrix-org/matrix-doc/issues/1500

### CODE REVIEW

File: unstyled_docs/spec/intro.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+  <body>
+    <!-- redirect with javascript if possible, to preserve the fragment. -->
+    <script>
```
Comment (ara4n) [inline unstyled_docs/spec/intro.html]: if you're insisting on writing xhtml 1(!) i think this has to have a `type="text/javascript"` technically.

**Comment by ara4n** [comment]
lgtm other than minor "i want to speak javascript, not vbscript" nit

File: unstyled_docs/spec/intro.html (hunk 1 above)
Comment (ara4n) [reply unstyled_docs/spec/intro.html]: in fact, it's worse - for xhtml 1 you'd have had to have done

```
<script type="text/javascript">
//<![CDATA[
...
//]]>
</script>
```

to be safe. surely just use html5 (i.e. dump the XML header, and have a DTD of `<!DOCTYPE html>` and then you can just use a nude `<script>` tag.

Comment (richvdh) [reply unstyled_docs/spec/intro.html]: ugh I hate this [x]html[12345] stuff. I just cargoculted the headers from somewhere else. will update

**Comment by ara4n** [comment]
lgtm

**Pull request merged**